addons/mrp/models/mrp_production.py

```diff
@@ -245,6 +243,12 @@ def _get_default_is_locked(self):
     show_produce = fields.Boolean(compute='_compute_show_produce', help='Technical field to check if produce button can be shown')
     show_produce_all = fields.Boolean(compute='_compute_show_produce', help='Technical field to check if produce all button can be shown')
     is_outdated_bom = fields.Boolean("Outdated BoM", help="The BoM has been updated since creation of the MO")
+    product_custom_attribute_value_ids = fields.One2many(
+        'product.attribute.custom.value', 'res_id',
```

Comment:
I agree that it's not the best way to name that variable but changing it will either impact sales or eitherthe js part of the configurator as they need to use the same fields names. So either we modify it in the mixin and everywhere in sale, either we separate the js parts ("product configurator" and `sale_product_configurator`), both ways are quite a lot of work.
What do you think is the best solution?